src/core/Jwt.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+class Jwt {
+  constructor (encodedJwt) {
+    this._encodedJwt = encodedJwt;
+
+    this._userId = null;
+    this._expiresAt = null;
+
+    this._decode();
+  }
+
+  get encodedJwt () {
+    return this._encodedJwt;
+  }
+
+  get userId () {
+    return this._userId;
+  }
+
+  get expiresAt () {
+    return this._expiresAt;
+  }
+
+  get expired () {
+    return Date.now() > this.expiresAt;
+  }
+
+  _decode () {
+    const  [, payloadRaw, ] = this._encodedJwt.split('.');
+
+    if (!payloadRaw) {
+      throw new Error('Invalid JWT format');
+    }
+    
+    let payload;
+    try {
+      payload = JSON.parse(new Buffer(payloadRaw, 'base64').toString());
```

Comment:
I missed that one, but `new Buffer` is deprecated, you should use Buffer.from instead:

```suggestion
      payload = JSON.parse(Buffer.from(payloadRaw, 'base64').toString());
```

Also, Buffer does not exist on browsers, does Babel correctly handle it?
If not, you may want to check if `Buffer` exists in the global scope and, if it doesn't, use btoa instead